apps/condo/domains/common/utils/tables.utils.ts

```diff
@@ -380,3 +382,12 @@ export const convertColumns = (
 export const getSorterMap: SorterMapType = (sorters) => {
     return Object.assign({}, ...sorters.map((sorter) => ({ [sorter.columnKey]: sorter.order })))
 }
+
+export const sorterUnitName: (a: Ticket, b: Ticket) => number = (a: Ticket, b: Ticket) => {
```

Comment:
bad function name. `sortUnitName`

Reply:
This is actually a sorter.
`unitNameSorter`